wolfssl/openssl/ssl.h

```diff
@@ -643,8 +643,7 @@ typedef WOLFSSL_X509_NAME_ENTRY X509_NAME_ENTRY;
 #define X509_V_FLAG_CRL_CHECK     WOLFSSL_CRL_CHECK
 #define X509_V_FLAG_CRL_CHECK_ALL WOLFSSL_CRL_CHECKALL
 
-#define X509_V_FLAG_PARTIAL_CHAIN 0x80000
-#define X509_V_FLAG_TRUSTED_FIRST 0
```

Comment:
Why was X509_V_FLAG_TRUSTED_FIRST removed here?

Reply:
misunderstanding -- fixed.  it's needed for gRPC.

Reply:
Its defined value is wrong. Also @douzzer and I discussed and decided only flags with wolfssl equivalents should be here.